src/main/java/seedu/bigpp/command/buildercommand/BuilderUnselectCommand.java

```diff
@@ -0,0 +1,51 @@
+package seedu.bigpp.command.buildercommand;
+
+import seedu.bigpp.command.Command;
+import seedu.bigpp.datastorage.DataStorage;
+import seedu.bigpp.exceptions.builderexceptions.BuilderIncorrectComponentException;
+import seedu.bigpp.exceptions.builderexceptions.BuilderMissingComponentException;
+import seedu.bigpp.exceptions.builderexceptions.BuilderNullComponentException;
+import seedu.bigpp.pc.PCList;
+import seedu.bigpp.ui.UI;
+
+public class BuilderUnselectCommand extends Command {
+
+    public BuilderUnselectCommand(String arguments) {
+        setArguments(arguments);
+    }
+
+    /**
+     * Change the Component of the PC to null
+     * @return component removed message
+     */
+    @Override
+    public String executeCommand(DataStorage dataStorage)
+            throws BuilderIncorrectComponentException, BuilderMissingComponentException, BuilderNullComponentException {
```

Comment:
Great use of including specific exceptions instead of generalising the exceptions